### PR TITLE
regions plugin: use isResizing and isDragging to filter events in region-updated listener

### LINKED ISSUE #1713: How to call onDrag & onResize function in wavesurefer regions ?
Hi all

I need regions `start and end `value for **region resize**.

How to call onDrag & onResize function in wavesurefer regions ?

REF : https://github.com/katspaugh/wavesurfer.js/issues/1555

cc - @katspaugh @MaxwellGover

### PR DESCRIPTION
**Pull request by aburai**
fixes #1713

### Short description of changes:

add regions props isResizing and isDragging to filter events in region-updated listener

### CODE REVIEW

File: src/plugin/regions.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ class Region {
                 : Number(params.end);
         this.resize =
             params.resize === undefined ? true : Boolean(params.resize);
+        this.isResizing = false;
         this.drag = params.drag === undefined ? true : Boolean(params.drag);
+        this.isDragging = false;
         this.loop = Boolean(params.loop);
```
Comment (thijstriemstra) [inline src/plugin/regions.js]: can you add some comment for both the isResizing and isDragging (here or elsewhere)?

File: src/plugin/regions.js [hunk 2]
```diff
@@ -121,7 +123,6 @@ class Region {
             );
         }
 
-        const width = this.wrapper.scrollWidth;
```
Comment (thijstriemstra) [inline src/plugin/regions.js]: why remove this?

Comment (aburai) [reply src/plugin/regions.js]: unused variable

File: src/plugin/regions.js (hunk 1 above)
Comment (aburai) [reply src/plugin/regions.js]: ok, i will do this. hope this solution works for the issue opener

**Pull request merged**